Sources/ClientRuntime/EventStream/DefaultMessageDecoderStream.swift

```diff
@@ -33,24 +33,22 @@ extension EventStream {
             }
 
             mutating public func next() async throws -> Event? {
```

Comment:
Simplified this loop a bit by moving the `while` condition to the end of the loop